papaparse.js

```diff
@@ -1165,20 +1170,19 @@ License: MIT
 			return value;
 		}
 
-		function applyHeaderAndDynamicTypingAndTransformation()
+		function applyHeaderAndDynamicTypingAndTransformation(isStep)
 		{
 			if (!_results || (!_config.header && !_config.dynamicTyping && !_config.transform))
 				return _results;
 
-			for (var i = 0; i < _results.data.length; i++)
+			function processRow(rowSource, i)
 			{
 				var row = _config.header ? {} : [];
 
-				var j;
-				for (j = 0; j < _results.data[i].length; j++)
+				function processField(value, j)
 				{
 					var field = j;
-					var value = _results.data[i][j];
+					// var value = _results.data[i][j];
```

Comment:
Why comenting?

Reply:
sorry, that was sloppy. I meant to remove it.